src/components/MatPassContent.tsx

```
@@ -372,115 +532,141 @@ export default function MatPassContent() {
+    (sum, item) => sum + item.price * item.quantity,
+    0
+  );
   const cartItemCount = cartItems.reduce((sum, item) => sum + item.quantity, 0);
 
   // Add to cart function - updated to add to both local cart and dispatch event
-  const addToCart = (id: number, name: string, price: number, isPass: boolean = false, isAcro: boolean = false, bundleTypeId?: string) => {
+  const addToCart = (
+    id: number,
+    name: string,
+    price: number,
+    isPass: boolean = false,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)